tools/dev-release.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh -xe
+# Release dev packages to PyPI
+
+version="0.0.0.dev$(date +%Y%m%d)"
+DEV_RELEASE_BRANCH="dev-release"
+# TODO: create a real release key instead of using Kuba's personal one
+RELEASE_GPG_KEY="${RELEASE_GPG_KEY:-148C30F6F7E429337A72D992B00B9CC82D7ADF2C}"
+
+# port for a local Python Package Index (used in testing)
+PORT=${PORT:-1234}
+
+# subpackages to be released
+SUBPKGS=${SUBPKGS:-"acme letsencrypt_apache letsencrypt_nginx letshelp_letsencrypt"}
+subpkgs_dirs="$(echo $SUBPKGS | sed s/_/-/g)"
+# letsencrypt_compatibility_test is not packaged because:
+# - it is not meant to be used by anyone else than Let's Encrypt devs
+# - it causes problems when running nosetests - the latter tries to
+#   run everything that matches test*, while there are no unittests
+#   there
+
+tag="v$version"
+mv "dist.$version" "dist.$version.$(date +%s).bak" || true
+git tag --delete "$tag" || true
+
+root="$(mktemp -d -t le.$version.XXX)"
+echo "Cloning into fresh copy at $root"  # clean repo = no artificats
+git clone . $root
```

Comment:
You should always build/release version X with the build/release scripts in the tree as of version X.  Otherwise we'll get horrible non-determinism where a change to those scripts will cause past versions to be very difficult to reproduce.